setup.py

```diff
@@ -11,20 +11,20 @@
       packages=['pipeline_tools'],
       install_requires=[
           'cromwell-tools',
-          'google-cloud-storage>=1.8.0',
-          'hca>=3.5.0',
-          'mock>=2.0.0',
-          'requests>=2.18.4',
-          'requests-mock>=1.4.0',
-          'setuptools_scm==2.0.0',
-          'tenacity>=4.10.0',
+          'google-cloud-storage>=1.8.0<2',
+          'hca>=3.5.0<4',
```

Comment:
Does this conflict with the other PR: https://github.com/HumanCellAtlas/pipeline-tools/pull/56/files#diff-2eeaed663bd0d25b7e608891384b7298R15 ?